app/handlers.py

```diff
@@ -43,19 +43,35 @@ async def get(self, request):
         }
 
 
-@static_routes.view(r'/' + View.valid_display_regions + '/cookies/')
+@static_routes.view(r'/' + View.valid_display_regions + '/cookies')
```

Comment:
I think we need to keep the trailing slashes on the routes everywhere, it looks like the app is currently set up to redirect non-trailing slash URLs to trailing slash but not vice versa, so the routes need to be set up with the trailing slash everywhere so that both work.